product_dimension/product.py

```diff
@@ -21,7 +21,7 @@
 
 
 class Product(models.Model):
-    _inherit = 'product.product'
+    _inherit = 'product.template'
```

Comment:
Hello, In v8.0 there are both "product.template" & "product.product". I need the product dimension in V9.0.
Should I make a PR from the actual v8.0 version of this module.
@pedrobaeza Why is it kept on purpose at variant level?